cimbar/cimbar.py

```diff
@@ -65,11 +59,19 @@ def get_deskew_params(level):
 
 
 def bits_per_op():
-    return BITS_PER_SYMBOL + BITS_PER_COLOR
+    return conf.BITS_PER_SYMBOL + BITS_PER_COLOR
 
 
-def _fountain_chunk_size(ecc=ECC, bits_per_op=bits_per_op(), fountain_blocks=FOUNTAIN_BLOCKS):
-    return int((155-ecc) * bits_per_op * 10 / fountain_blocks)
+def num_cells():
+    return conf.CELL_DIM_Y*conf.CELL_DIM_X - (conf.MARKER_SIZE_X*conf.MARKER_SIZE_Y * 4)
```

Comment:
The grid is still square (for now?) but the number of cells might not be (in the case where we forgo padding in one dimension).